wgengine/magicsock/magicsock.go

```diff
@@ -567,6 +567,108 @@ func (c *Conn) SetNetInfoCallback(fn func(*tailcfg.NetInfo)) {
 	}
 }
 
+// peerForIP returns the Node in that nm that's responsible for
+// handling the given IP address.
+func peerForIP(nm *controlclient.NetworkMap, ip netaddr.IP) (n *tailcfg.Node, ok bool) {
+	wgIP := wgcfg.IP{Addr: ip.As16()}
```

Comment:
Comment something like: Check for exact matches before looking for subnet matches. (Only more accurate. I'm guessing that that is what this code does.)

Reply:
Done.